charts/k0s/templates/coredns.yaml

```diff
@@ -132,6 +132,7 @@ data:
         spec:
           priorityClassName: "system-cluster-critical"
           serviceAccountName: coredns
+          automountServiceAccountToken: true
```

Comment:
I noticed that in this file you have not set the `automountServiceAccountToken: false` on the ServiceAccount